lib/sqlalchemy/ext/asyncio/result.py

```diff
@@ -114,7 +116,7 @@ def t(self) -> AsyncTupleResult[_TP]:
         """
         return self  # type: ignore
 
-    def tuples(self) -> AsyncTupleResult[_TP]:
+    def tuples(self) -> AsyncTupleResult[Tuple[Unpack[_Ts]]]:
```

Comment:
**Federico Caselli** (CaselIT) wrote:

could be deprecated. Same as `t`

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986